components/mediation-admin/org.wso2.carbon.rest.api/src/main/java/org/wso2/carbon/rest/api/RestApiAdminUtils.java

```diff
@@ -260,7 +271,7 @@ public static void persistApi(API api) throws APIException {
     }
 
     public static boolean isNullOrEmpty(String s) {

```

Comment:
I saw many locations we do something like 
`someString != null && !someString.isEmpty()`
cant we use this method on those locations?

Reply:
Yes, we can do that. The reason I did not change the code because it was not related to this fix. If we are to do this, we will have to refactor and address the issues in a separate commit.

Reply:
okay, got it :+1: